bench-wasm.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
+# Default benchmark name
+BENCH_NAME=""
+
+# Parse the optional --bench=<NAME> argument
+for i in "$@"; do
+    case $i in
+        --bench=*)
+        BENCH_NAME="${i#*=}"
+        shift # Remove --bench=<NAME> from processing
+        ;;
+        *)
+        # unknown option
+        ;;
+    esac
+done
+
+# Throw an error if --bench was not provided
+if [ -z "$BENCH_NAME" ]; then
+    echo "Error: You must specify a benchmark using --bench=<NAME>"
+    exit 1
+fi
+
+# Function to delete any old .wasm files related to the current benchmark
+cleanup_old_wasm_files() {
+    WASM_FILES=$(find target/wasm32-wasi/release/deps/ -type f -name "*$BENCH_NAME*.wasm")
+    
+    if [ -n "$WASM_FILES" ]; then
+        echo "Cleaning up old WASM files for benchmark '$BENCH_NAME'..."
+        rm -f $WASM_FILES
+    else
+        echo "No old WASM files found for benchmark '$BENCH_NAME'."
+    fi
+}
+
+# Call the cleanup function
+cleanup_old_wasm_files
```

Comment:
Can you add `cd $(git rev-parse --show-toplevel)` or something like that? Maybe another argument that would specify from which directory to execute, so we have to pass `.`? This script is assumed to be called /exactly/ from a directory we need it to run into, and I think it's not a very obvious UX and can potentially erase some files you didn't expect it to touch.